gapic-generator-cloud/Rakefile

```diff
@@ -37,6 +37,56 @@ task :ci do
   Rake::Task[:test].invoke
 end
 
+desc "Update the proto binary dumps which are used to generate templates"
+task :bin do
+  Rake::Task["bin:templates"].invoke
+end
+
+namespace :bin do |bin_namespace|
+  desc "Update the proto binary dumps for the templates"
+  task :templates do |templates_task|
+    bin_namespace.tasks.reject { |t| t == templates_task }.each do |t|
```

Comment:
I think it would be easier just to define an array of the libraries like

```ruby
libraries = [:language_v1, :language_v1beta1, ...]
```

Then you can use that here rather than trying to introspect the namespace and remove special tasks. And when defining the tasks below, you can just loop over the array.

Reply:
:+1: